library/src/main/java/com/bumptech/glide/load/resource/bitmap/DrawableTransformation.java

```diff
@@ -58,6 +58,7 @@ public DrawableTransformation(Transformation<Bitmap> wrapped, boolean isRequired
         return resource;
       }
     }
+    // TODO bitmapResourceToTransform can be null, what to do?
```

Comment:
It will be non-null if you reach this point, I believe you can remove this todo.

Reply:
@sjudd Oh you're right, thanks!